modules/local/check_torch_model.nf

```diff
@@ -7,18 +7,21 @@ process CHECK_TORCH_MODEL {
     
     input:
     tuple path(original_csv), path(model),  path(experiment_config), path(ray_tune_config)
+    each path(initial_weights)
```

Comment:
how does each behave on an empty list gatherted through path()? does it return none?

Reply:
because this line handles it : `def args_initial_weights = initial_weights ? "--initial_weights $initial_weights" : ""`